samples/bookinfo/src/build-services.sh

```diff
@@ -44,29 +45,29 @@ pushd "$SCRIPTDIR/reviews"
+    docker build --pull -t "$PREFIX/examples-bookinfo-reviews-v3:${VERSION}" -t "$PREFIX/examples-bookinfo-reviews-v3:latest" --build-arg service_version=v3 \
 	   --build-arg enable_ratings=true --build-arg star_color=red .
   popd
 popd
 
 pushd "$SCRIPTDIR/ratings"
-  docker build --pull -t "istio/examples-bookinfo-ratings-v1:${VERSION}" -t istio/examples-bookinfo-ratings-v1:latest --build-arg service_version=v1 .
-  docker build --pull -t "istio/examples-bookinfo-ratings-v2:${VERSION}" -t istio/examples-bookinfo-ratings-v2:latest --build-arg service_version=v2 .
-  docker build --pull -t "istio/examples-bookinfo-ratings-v-faulty:${VERSION}" -t istio/examples-bookinfo-ratings-v-faulty:latest --build-arg service_version=v-faulty .
-  docker build --pull -t "istio/examples-bookinfo-ratings-v-delayed:${VERSION}" -t istio/examples-bookinfo-ratings-v-delayed:latest --build-arg service_version=v-delayed .
-  docker build --pull -t "istio/examples-bookinfo-ratings-v-unavailable:${VERSION}" -t istio/examples-bookinfo-ratings-v-unavailable:latest --build-arg service_version=v-unavailable .
-  docker build --pull -t "istio/examples-bookinfo-ratings-v-unhealthy:${VERSION}" -t istio/examples-bookinfo-ratings-v-unhealthy:latest --build-arg service_version=v-unhealthy .
+  docker build --pull -t "$PREFIX/examples-bookinfo-ratings-v1:${VERSION}" -t "$PREFIX/examples-bookinfo-ratings-v1:latest" --build-arg service_version=v1 .
+  docker build --pull -t "$PREFIX/examples-bookinfo-ratings-v2:${VERSION}" -t "$PREFIX/examples-bookinfo-ratings-v2:latest" --build-arg service_version=v2 .
+  docker build --pull -t "$PREFIX/examples-bookinfo-ratings-v-faulty:${VERSION}" -t "$PREFIX/examples-bookinfo-ratings-v-faulty:latest" --build-arg service_version=v-faulty .
+  docker build --pull -t "$PREFIX/examples-bookinfo-ratings-v-delayed:${VERSION}" -t "$PREFIX/examples-bookinfo-ratings-v-delayed:latest" --build-arg service_version=v-delayed .
+  docker build --pull -t "$PREFIX/examples-bookinfo-ratings-v-unavailable:${VERSION}" -t "$PREFIX/examples-bookinfo-ratings-v-unavailable:latest" --build-arg service_version=v-unavailable .
+  docker build --pull -t "$PREFIX/examples-bookinfo-ratings-v-unhealthy:${VERSION}" -t "$PREFIX/examples-bookinfo-ratings-v-unhealthy:latest" --build-arg service_version=v-unhealthy .
 popd
 
 pushd "$SCRIPTDIR/mysql"
-  docker build --pull -t "istio/examples-bookinfo-mysqldb:${VERSION}" -t istio/examples-bookinfo-mysqldb:latest .
+  docker build --pull -t "$PREFIX/examples-bookinfo-mysqldb:${VERSION}" -t "$PREFIX/examples-bookinfo-mysqldb:latest" .
 popd
 
 pushd "$SCRIPTDIR/mongodb"
-  docker build --pull -t "istio/examples-bookinfo-mongodb:${VERSION}" -t istio/examples-bookinfo-mongodb:latest .
+  docker build --pull -t "$PREFIX/examples-bookinfo-mongodb:${VERSION}" -t "$PREFIX/examples-bookinfo-mongodb:latest" .
```

Comment:
Replace `$PREFIX` with `${PREFIX}` in this file at several locations for consistency.